qemu/tests/cfg/usb_device_check.cfg

```diff
@@ -85,3 +85,18 @@
         - input_devices_under_two_tier_hub:
             no usb-ehci
             usb_topology = '{"usb-hub":2,"usb-mouse":1,"usb-kbd":1,"usb-tablet":1}'
+        - max_devices_under_one_hub:
+            no usb-ehci
+            usb_topology = '{"usb-hub":1,"usb-mouse":8}'
+        - usb_mouse:
```

Comment:
For this scenario, there is step for checking dmesg info of usb device, could you please show me it? I have not found it.

Thanks.

Reply:
I'd consider they are same. So I only use `lsusb` here.